bleachbit/FileUtilities.py

```diff
@@ -92,31 +92,39 @@ def __random_string(length):
 
 
 def bytes_to_human(bytes_i):
-    """Display a file size in human terms (megabytes, etc.) using SI standard"""
+    """Display a file size in human terms (megabytes, etc.) using preferred standard (SI or IEC)"""
 
-    storage_multipliers = {1000 ** 5: 'PB', 1000 ** 4: 'TB',
-                           1000 ** 3: 'GB', 1000 ** 2: 'MB', 1000: 'kB', 1: 'B'}
+    if bytes_i < 0:
+        return '-' + bytes_to_human(-bytes_i)
+
+    from Options import options
+    if options.get('units_iec'):
```

Comment:
Would you please add at least a basic test to tests/TestFileUtilities.py? It doesn't have to cover the round trip. It could be a new test function just for IEC.